exercises/sending-signals-client/README.md

```diff
@@ -0,0 +1,62 @@
+## Exercise #1: Sending a Signal from the Client
+
+This exercise shows how to send a Signal from just the Client and not from another Workflow.
+
+During this exercise, you will:
+
+- Define and handle a Signal
+- Retrieve a handle on the Workflow to Signal
+- Send a Signal from the Client
+
+In this part of the exercise, we have removed the `fulfillOrderWorkflow` Workflow, since this exercise is just for sending a Signal from a Client, and not from another Workflow.
```

Comment:
If this is supposed to be Exercise #1 after all (I think just the main readme for this repository needs to be updated, since the numbering in this file is consistent with the content repo now), this part doesn't make sense -- it's a forward reference explaining part of the next Exercise.